src/pandapipes/create.py

```diff
@@ -528,8 +519,8 @@ def create_pipe_from_parameters(net, from_junction, to_junction, length_km, diam
     return index
 
 
-def create_valve(net, from_junction, to_junction, diameter_m, opened=True, loss_coefficient=0,
-                 name=None, index=None, type='valve', **kwargs):
+def create_valve(net, from_junction, to_junction, diameter_m, opened=True, loss_coefficient=0, name=None, index=None,
```

Comment:
Is it really possible to change the line lengths here? Maybe we should start thinking about formatters or rules in codacy that check these specifics.